test/validate/service_territory_test.py

```diff
@@ -30,34 +32,25 @@ def test_minmax_rows(
     Args:
         pudl_out_orig: A PudlTabl output object.
         live_dbs: Whether we're using a live or testing DB.
-        expected_rows: Expected number of rows that the dataframe should
-            contain when all data is loaded and is output without aggregation.
-        df_name: Shorthand name identifying the dataframe, corresponding
-            to the name of the function used to pull it from the PudlTabl
-            output object.
+        expected_rows: Expected number of rows that the dataframe should contain when
+            all data is loaded and is output without aggregation.
+
+        df_name: Shorthand name identifying the dataframe, corresponding to the name of
+            the function used to pull it from the PudlTabl output object.
     """
     if not live_dbs:
         pytest.skip("Data validation only works with a live PUDL DB.")
-    _ = (
-        pudl_out_orig.__getattribute__(df_name)()
-        .pipe(
-            pv.check_min_rows, expected_rows=expected_rows, margin=0.0, df_name=df_name
-        )
-        .pipe(
-            pv.check_max_rows, expected_rows=expected_rows, margin=0.0, df_name=df_name
-        )
-    )
+    assert pudl_out_orig.__getattribute__(df_name)().shape[0] == expected_rows
```

Comment:
this seems totally fine.. but why not keep doing the standard thing? i get that we've long transitioned to always using a 0% margin and therefor this test for a long long time has really been a `test_expected_rows`... but still. absolutely a nit, but it always feels bad to slowly diverge away from something in a way that makes it harder to change it in a systematic way.

Reply:
I used the shape attribute because the normal way that we do it counts up the number of rows twice, (once for min, another for max) but `df.shape` doesn't require any calculation, and this dataframe has millions of rows, and I think we'll do something more like this going forward (as with reading the number of rows from the Parquet metadata, where it can be found without doing any computation.

Reply:
yea this seems like a much better method. i like what you are doing here, it is just a divergence from the standard process which imo is confusing to our future selves and more error prone/harder to track down future fixes. can you make a new smol issue which is a cleaning up of the min/max row validation tests with this as the suggested methodology and a note that when that happens we should make sure that these more one-off versions should be swooped up into that new methodology?